builds/misc/templates/image-linux.yaml

```diff
@@ -88,9 +88,9 @@ steps:
   - script: |
       scripts/linux/buildManifest.sh \
         -r '$(registry.address)' \
-        -n microsoft \
-        -t 'edge-agent/docker/manifest.yaml.template' \
-        -v '$(version)' \
+        -n '${{ parameters.namespace }}' \
+        -t '${{ parameters.manifestTemplate }}' \
+        -v '${{ parameters.version }}' \
         --tags '$(tags)'
-    displayName: Build Edge Agent Manifest
```

Comment:
Did this cause issues?

Reply:
Ultimately, no. It meant that the manifest wasn't published to the microsoft namespace in EdgeRelease ACR for the non-edge-agent images. But when we move images from microsoft to public namespace to stage them to MCR, we build the manifest again anyway, and that step doesn't use this template. So what got published to MCR was correct.